src/foam/dao/EasyDAO.js

```diff
@@ -162,6 +188,13 @@ if ( getContextualize() ) {
     build();
 }
 
+if ( getOrder() != null &&
+     getOrder().length > 0 ) {
+  for ( foam.core.PropertyInfo prop : getOrder() ) {
+    delegate = delegate.orderBy(prop);
```

Comment:
This will just order by the last property. You want a CompoundComparator, which we currently only support in JS, but we should fix/implement that. Will also require work on MDAO to support.

Reply:
Presently serviceScripts only have one orderBy so this will work for now and we can add support in another PR.